test/test_nestedtensor.py

```diff
@@ -3270,6 +3293,25 @@ def test_unbind(self, device):
             for i, t in enumerate(out):
                 self.assertEqual(t, tensor_list[i])
 
+    def test_layer_norm(self, device):
+        a = torch.randn(2, 3, requires_grad=True, dtype=torch.float64, device=device)
+        b = torch.randn(3, 3, requires_grad=True, dtype=torch.float64, device=device)
+        c = torch.randn(4, 3, requires_grad=True, dtype=torch.float64, device=device)
```

Comment:
FYI, we recently added this new helper so you can do something like `ts = self._get_list_for_jagged_tensor(((2, 3, 4), 3), device=device, requires_grad=True)`